enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -74,4 +75,28 @@ public void removeUserFromRole( @Name( "username" ) String username, @Name( "rol
         }
         shiroSubject.getRoleManager().removeUserFromRole( username, roleName );
     }
+
+    @PerformsDBMS
+    @Procedure( "dbms.suspendUser" )
+    public void suspendUser( @Name( "username" ) String username ) throws IOException, ConcurrentModificationException
```

Comment:
We don't want to expose the `ConcurrentModificationException` in procedures (see comments in `FileUserRealm`).